planning/behavior_path_planner/src/scene_module/side_shift/side_shift_module.cpp

```diff
@@ -191,7 +191,8 @@ void SideShiftModule::updateData()
 
   lanelet::ConstLanelet current_lane;
   if (!route_handler->getClosestLaneletWithinRoute(reference_pose.pose, &current_lane)) {
-    RCLCPP_ERROR(getLogger(), "failed to find closest lanelet within route!!!");
+    RCLCPP_ERROR_THROTTLE(
+      get_logger(), *get_clock(), 5000, "failed to find closest lanelet within route!!!");
```

Comment:
This `get_logger()` should be getLogger() or rclcpp::get_logger().

Reply:
Other modules has same codes Could you fix these?

Reply:
Thank you, Actually, now I'm making new pkg for checking route and it might affect this PR.